examples/seldon_custom_pytorch_deployment/README.md

```diff
@@ -0,0 +1,502 @@
+[Install kubectl and configure cluster access](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl)
+
+To view the list of contexts for kubectl, run the following command
+
+```bash
+kubectl config current-context
+```
+
+Set up a namespace for ZenML Seldon Core workloads:
+
+```bash
+kubectl create ns zenml-workloads
+```
+
+Extract the URL where the Seldon Core model server exposes its prediction API, e.g.:
+
+```bash
+export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+Give Seldon Core access to the GCP artifact store in the configured namespace.
+For this we need first to create a service-account key and have it as local json file. First make sure that you have [SA-NAME]@[PROJECT-ID].iam.gserviceaccount.com service account created in the gcloud console that have sufficient permissions to access the bucket where model will be stored (i.e. Storage Object Admin).
+
+Generate the service-account key using gcloud.
+```bash
+gcloud iam service-accounts keys create gcloud-application-credentials.json --iam-account [SA-NAME]@[PROJECT-ID].iam.gserviceaccount.com
+```
+
+Give Seldon Core access to the GCP artifact store in the zenml-workloads namespace:
+
+```bash
+kubectl -n zenml-workloads create secret generic seldon-rclone-secret \
+    --from-literal=RCLONE_CONFIG_GS_TYPE='google cloud storage' \
+    --from-literal=RCLONE_CONFIG_GS_ANONYMOUS='false' \
+    --from-literal=RCLONE_CONFIG_GS_SERVICE_ACCOUNT_CREDENTIALS="$(cat <path-to-json-key>)"
+```
+
+The name of the created secret also needs to be passed to the `run.py` example
+launcher script via its `--secret` command line argument.
+
+NOTE: this is based on the assumption that Seldon Core is running in an GKE
+cluster that runs in the same project where container regsitry is defined otherwise
+the permission should be granted to GKE instances to access the registry. 
+More informations can be found at 
+[the official GCP documentation](https://cloud.google.com/artifact-registry/docs/integrate-gke)
+
+Configuring the stack can be done like this:
+
+```
+zenml integration install gcp seldon
+zenml model-deployer register seldon_gcp --type=seldon \
+  --kubernetes_context=gke_cluster-1 --kubernetes_namespace=zenml-workloads \
+  --base_url=http://34.139.128.148
+zenml artifact-store register gcp --type gcp --path gs://mybucket
+zenml container-registry register gcp_registry --type=default --uri=gcr.io/container-registry
+zenml stack register gcp_stack_seldon -m default -a gcp -o default -d seldon_gcp -c gcp_registry
+zenml stack set gcp_stack_seldon
+```
+
+#### Full GCP stack
+
+This stack has all components running in the GCP cloud:
+
+* an GCP artifact store
+* a Kubeflow orchestrator installed in an AWS EKS Kubernetes cluster
+* a metadata store that uses the same database as the Kubeflow deployment as
+a backend
+* an GCP container registry (artifact registry)
```

Comment:
What does this mean? An artifact store or an artifact registry? And do you mean we have both an artifact store and a container registry together on GCP?